lib/pengine/rules.c

```diff
@@ -312,7 +312,7 @@ pe_evaluate_rules(xmlNode *ruleset, GHashTable *node_hash, crm_time_t *now,
     return pcmk__evaluate_rules(ruleset, &rule_input, next_change);
 }
 
-gboolean
+static gboolean
 pe_test_rule(xmlNode *rule, GHashTable *node_hash, enum rsc_role_e role,
```

Comment:
Could drop library prefix for clarity of scope, not important. Would have to rename a bit more since there's a `test_rule()` that we can't drop yet.